cmd/celestia/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"os"
+
+	logging "github.com/ipfs/go-log/v2"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(fullCmd, lightCmd)
+}
+
+func main() {
+	err := run()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func run() error {
+	// TODO(@Wondertan): In practice we won't need all INFO loggers from IPFS/libp2p side
+	//  so we would need to turn off them somewhere in `logs` package.
+	logging.SetAllLoggers(logging.LevelInfo)
+	return rootCmd.Execute()
+}
+
+var rootCmd = &cobra.Command{
+	Use: "celestia [subcommand]",
```

Comment:
```suggestion
	Use: "celestia [  full  ||  light  ] [subcommand]",
```

Reply:
Reason being, it makes it more clear that celestia can only be used if `full` or `light` mode is defined. We can update this later when we implement more node types.